deployment/media-insights-stack.yaml

```diff
@@ -692,6 +685,12 @@ Resources:
         EmailSendingAccount: 'COGNITO_DEFAULT'
       AutoVerifiedAttributes: ['email']
 
+  MieCognitoDomain:
```

Comment:
This goes inline with the above comment - helps reduce the work required when later setting up kibana.